Mathlib/Data/Set/Subset.lean

```diff
@@ -112,8 +112,10 @@ lemma image_val_union_self_left_eq : ↑D ∪ A = A :=
   union_eq_right.2 image_val_subset
 
 @[simp]
-lemma cou_inter_self_right_eq_coe : A ∩ ↑D = ↑D :=
+lemma image_val_inter_self_right_eq_coe : A ∩ ↑D = ↑D :=
```

Comment:
Wow, this name was a mess, but your fix is still not quite how I read it from left-to-right. How about this?
```suggestion
lemma inter_image_val_eq_right : A ∩ ↑D = ↑D :=
```